src/quo2/components/notifications/toast.cljs

```diff
@@ -51,8 +51,8 @@
     [i18n/label :t/undo]]])
 
 (defn- toast-container
-  [{:keys [left middle right]}]
-  [rn/view {:style {:padding-left 12 :padding-right 12}}
+  [{:keys [left middle right container-style]}]
+  [rn/view {:style (merge {:padding-left 12 :padding-right 12} container-style)}
```

Comment:
imo, i like the use of "container-style" as it avoid adding a wrapping View component around components. Maybe we can standardise this and include it in the best practices file??
@erikseppanen, @ilmotta , @flexsurfer et all wdyt? 🤔

Reply:
Sorry @J-Son89, I didn't quite understand the problem `container-style` is solving, could you explain again, please?

Reply:
@ilmotta I think this is what @J-Son89  is referring to:
```
[rn/view
        {:style {:position :absolute
                 :right    16}}
        [quo/icon :i/check
         {:size 20 :color (colors/theme-colors colors/primary-50 colors/primary-60)}]]
```
Notice how I had to add a view in order to add `View` styling to icon component which but by default only accepts `Text` styling

Reply:
It avoids the use in status_im2 code where we do


Yes exactly, it's to avoid always needing a wrapper component when using quo2 components.

E.g
No container style prop
'''
[rn/view { :margin-left  10
           :margin-right 10}
  [quo2/component ...]]
'''
Container style prop
'''
 [quo2/component {:container-style { :margin-left  10
           :margin-right 10}}]
'''

Reply:
Cool, it makes complete sense to me, thanks guys. Just one minor detail, sometimes using the word `style` is better because there's no container around the component, so the rule should have some exceptions IMO.

Reply:
Yep, container is just to imply that those styles would apply to the outer most part of the component. styles could be any part of the component.